CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Removed:
 
 Added:
 * Add burn rate configs to the unstable `chronosphere_slo` resource.
+* Updated unstable API spec to latest version.
```

Comment:
since a new swagger spec doesn't have any user-facing changes, no change log is needed.

The CI check can be skipped with the `skip-changelog` label on the PR

Reply:
Ah... Good to know. I'll fix that!